build_docker.sh

```diff
@@ -124,9 +126,17 @@ fi
 
 echo "Building image to tag ${REPO_PRJ}:${GITHUB_TAG}..."
 if [ -n "${IS_PUSH}" ]; then
-    docker build -t ${REPO_PRJ}:${GITHUB_TAG} --build-arg RELEASE=${RELEASE}  --squash .
+    if [ -n "${LITE}" ]; then
+        docker build -t ${REPO_PRJ}:${GITHUB_TAG} --build-arg RELEASE=${RELEASE} --target gatk-lite --squash .
```

Comment:
What's the name of the lite docker?  I think we have to modify the -t instruction with `-lite` on the end?

Reply:
I've been thinking of this like, if it's a lite docker, we just stick "lite-" at the beginning of the tag.  That's what I'm doing for the generated tags in build_docker_remote.sh.  Here, though, you can only specify a tag explicitly, and I figured it made the most sense to just use exactly the tag that's provided, since whoever is running this script has to choose explicitly to generate the lite docker